doc/cephfs/administration.rst

```diff
@@ -92,6 +92,14 @@ The CephX IDs authorized to the old file system name need to be reauthorized
 to the new name. Any on-going operations of the clients using these IDs may be
 disrupted. Mirroring is expected to be disabled on the file system.
 
+::
+
+    fs swap <fs1-name> <fs2-name> [--swap-fscids] [--yes-i-really-mean-it]
+
+Swaps names of two Ceph file sytems and updates the application tags on all
+pools of both FSs accordingly. Passing ``swap-fscids`` swaps FSCIDs as well.
```

Comment:
> Passing ``swap-fscids`` swaps FSCIDs as well.

We should explain why that's an option and why it's dangerous. (Hint: any tool only tracking the fscid may become confused. CSI requires the FSCID to be preserved.)

Reply:
ping @rishabh-d-dave